### ISSUE TITLE
[Bug]: Integration doesn't work after upgrade HA Core to V2022.8.7

### ISSUE THREAD

**Issue by johanvh83** [wlcrs/huawei_solar#123]
### System Health details

Home Assistant 2022.8.7
Supervisor 2022.08.3
Operating System 8.4
Frontend 20220802.0 - latest

### Huawei Solar Setup

Inverter: SUN2000-4.6KTL-L1 - Firmware: V200R001C00SPC115
Single phase
Battery type: LUNA 2000 10 kWh
Power meter connected

### Describe the issue

Since update from 2022.8.6 to 2022.8.7 I have below error (see logs).
I had a full backup from yesterday so put back to V2022.8.6 but the problem remains.
I even restored to a full backup 48 hours old but no solution.
No idea how to resolve this problem.
I tried to delete the integration and to re-install it but I'm not able to delete it...
Finaaly I deleted manually the 'huawei_solar' folder in  the 'custom components' folder and redownload the respository with HACS and re-started HA but also that action didn't help.
I really don't know how to proceed to resolve this problem.

### Relevant debug logs

```text
Logger: homeassistant.setup
Source: setup.py:184
First occurred: 11:18:51 AM (1 occurrences)
Last logged: 11:18:51 AM

Setup failed for custom integration huawei_solar: Requirements for huawei_solar not found: ['huawei-solar==2.0.0rc5'].


Logger: homeassistant.util.package
Source: util/package.py:98
First occurred: 11:18:44 AM (3 occurrences)
Last logged: 11:18:51 AM

Unable to install package huawei-solar==2.0.0rc5: ERROR: Could not find a version that satisfies the requirement huawei-solar==2.0.0rc5 (from versions: 0.0.1.linux-x86_64, 0.0.2.linux-x86_64, 0.0.3.linux-x86_64, 0.0.4.linux-x86_64, 0.0.5.linux-x86_64, 0.0.6.linux-x86_64, 0.0.7.linux-x86_64, 0.0.8.linux-x86_64, 0.0.9.linux-x86_64, 0.0.10.linux-x86_64, 0.0.11.linux-x86_64, 0.1.0b2.linux-x86_64, 0.1.0b4.linux-x86_64, 0.1.0.linux-x86_64, 0.1.1.linux-x86_64, 1.0.1.linux-x86_64, 1.1.0b1.linux-x86_64, 1.1.0.linux-x86_64, 1.2.0.linux-x86_64, 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.1.0b2, 0.1.0b4, 0.1.0, 0.1.1, 1.0.1, 1.1.0b1, 1.1.0, 1.2.0, 2.0.0, 2.0.1, 2.0.2, 2.1.0, 2.1.1, 2.1.2, 2.1.3) ERROR: No matching distribution found for huawei-solar==2.0.0rc5
```

2022-08-28 11:48:59.534 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration huawei_solar which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-08-28 11:48:59.546 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration huawei_solar which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-08-28 11:49:33.132 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration huawei_solar: Requirements for huawei_solar not found: ['huawei-solar==2.0.0rc5'].

**Comment by wlcrs** [comment]
Please upgrade your integration to the latest version. You are running an outdated and unsupported version.

**Comment by johanvh83** [comment]
Thanks for your comment. Even when updateing to beta V1.1.3b it's not resolved.

**Comment by johanvh83** [comment]
I don't believe the problem is related to HA versions or your integrations versions.
Any idea where this comes from: Unable to install package huawei-solar==2.0.0rc5

**Comment by johanvh83** [comment]
Hi wlcrs, The problem is resolved, finally, after many hours.
1 year ago I started after the solar installation with the EmilV2 integration. Since I use your integration I just had removed the EmilV2 YAML configuration and renamed the EmilV2 folder in the custom_components folder. For some reason this occured the problem with your integration.
I removed the EmilV2 folder and restarted HA. Your integration starts now without any problem!
